database_setup.sql

```diff
@@ -1,11 +1,10 @@
-CREATE DATABASE IF NOT EXISTS groot_caffeine_service_dev;
```

Comment:
Caffeine was always groot-merch-service (the database name specified was probably named this at one point before we made the change to rename caffeine to merch).